.github/workflows/pr-conflict-detection.yml

```
@@ -0,0 +1,302 @@
+            echo "[FETCH] PRs to check:"
+            cat pr_numbers.txt | while read pr; do
+              echo "[FETCH]   - PR #${pr}"
+            done
+          fi
+          echo "[FETCH] ========================================"
+
+      - name: Check single PR for conflicts
+        if: github.event_name == 'pull_request_target'
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.10)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)